DataLink.tex

```diff
@@ -707,9 +707,10 @@ \subsection{Successful Requests}
 with some additional details below.
 
 Unless the incoming request included a RESPONSEFORMAT parameter requesting
-a different format, the content-type header of the response MUST be
-``application/x-votable+xml'' with the
-``content'' parameter set to ``datalink'',
+a different format, the content-type header of the response MAY be any value
```

Comment:
Agree with Markus, MAY doesn't make sense here, should be SHOULD (or possibly MUST)